app/src/main/java/com/codingblocks/cbonlineapp/database/AppDatabase.kt

```diff
@@ -22,13 +22,14 @@ import com.codingblocks.cbonlineapp.database.models.RunModel
 import com.codingblocks.cbonlineapp.database.models.RunPerformance
 import com.codingblocks.cbonlineapp.database.models.SectionContentHolder
 import com.codingblocks.cbonlineapp.database.models.SectionModel
+import com.codingblocks.cbonlineapp.database.models.CodeChallengeModel
 
 @Database(
     entities = [CourseModel::class, SectionModel::class, ContentModel::class, InstructorModel::class, Notification::class,
         CourseWithInstructor::class, DoubtsModel::class, NotesModel::class, RunModel::class,
         JobsModel::class, SectionContentHolder.SectionWithContent::class, BookmarkModel::class,
-        CommentModel::class, RunAttemptModel::class, RunPerformance::class, PlayerState::class
-    ], exportSchema = true, version = 29
+        CommentModel::class, RunAttemptModel::class, RunPerformance::class, PlayerState::class,CodeChallengeModel::class
+    ], exportSchema = true, version = 30
```

Comment:
You've added 3 of the database schema can you just add one of them that is 30.json

Reply:
Also, don't resolve my comments as it makes more difficult to keep track of the changes

Reply:
Done sir, removed 31 and 32.json